File: Angular/satva/src/app/consumer.ts
import { Address } from "./address";

export class Consumer {
    private consumerId: number = 0;
    public get consumerid(): number {
        return this.consumerId;
    }
    public set consumerid(value: number) {
        this.consumerId = value;
    }
     firmName: string = "";
    public get firmname(): string {
        return this.firmName;
    }
    public set firmname(value: string) {
        this.firmName = value;
    }
	private gstin: string = "";
    public get GSTiN(): string {
        return this.gstin;
    }
    public set GSTiN(value: string) {
        this.gstin = value;
    }
	private emailId: string = "";
    public get emailid(): string {
        return this.emailId;
    }
    public set emailid(value: string) {
        this.emailId = value;
    }
	private password: string = "";   
    public get passWord(): string {
        return this.password;
    }
    public set passWord(value: string) {
        this.password = value;
    }
    private firmAddress:Address  = new Address();
    public get add(): Address {
        return this.firmAddress;
    }
    public set add(value: Address) {
        this.firmAddress = value;
    }
}

File: SpringBoot/demo/src/main/java/com/example/rps/beans/buy/Consumer.java
package com.example.rps.beans.buy;

import com.example.rps.beans.Address;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Consumer 
{
	private int consumerId;
	private String firmName;
	private String emailId;
	private String password;
	private Address firmAddress;
	private String gstin;
	public String getGstin() {
		return gstin;
	}
	public void setGstin(String gstin) {
		this.gstin = gstin;
	}
	public String getFirmName() {
		return firmName;
	}
	public void setFirmName(String firmName) {
		this.firmName = firmName;
	}
	public String getEmailId() {
		return emailId;
	}
	public void setEmailId(String emailId) {
		this.emailId = emailId;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public Address getFirmAddress() {
		return firmAddress;
	}
	public void setFirmAddress(Address firmAddress) {
		this.firmAddress = firmAddress;
	}
	
	public int getConsumerId() {
		return consumerId;
	}
	public void setConsumerId(int consumerId) {
		this.consumerId = consumerId;
	}

}
